daemon/cmd/state.go

```diff
@@ -46,6 +46,19 @@ func (d *Daemon) WaitForEndpointRestore(ctx context.Context) error {
 	return nil
 }
 
+func (d *Daemon) WaitForInitialEnvoyPolicy(ctx context.Context) error {
```

Comment:
Why is it `WaitForInitialEnvoyPolicy` if the channel is not specifically about Envoy (or at least that's what the channel name suggests)? I think clarifying this will clear up any confusion to future readers.

Reply:
Full policy regeneration consists of both bpf policy map update and Envoy policy update (ignoring the DNS policy update, which is done before either of these when the selector policy is updated). This PR updates the initial selector policy and envoy policy (based on the updated selector policy) before queueing the endpoint regeneration. Possible bpf policy map updates are done after the event is dequeued, in the "normal" code path.

Thus `WaitForInitialEnvoyPolicy` is quite literally about the initial Envoy policy being done, so that our xDS server can start serving xDS resources to Envoy.

Reply:
Ok that's fine with me, could we rename the channel to match?